Quaver.Shared/Screens/Select/SelectScreen.cs

```diff
@@ -542,5 +573,184 @@ public void SelectRandomMap()
                     break;
             }
         }
+
+        /// <summary>
+        ///     Used to start the deleting process.
+        ///
+        ///     woo now AiAe will stop telling me to add this.
+        /// </summary>
+        /// <param name="type"></param>
+        private void DeleteSelected()
+        {
+            var view = View as SelectScreenView;
+            var type = view.ActiveContainer;
+            var filePath = string.Empty;
+
+            var selectedMapset = AvailableMapsets[view.MapsetScrollContainer.SelectedMapsetIndex];
+
+            if (AvailableMapsets.Count == 0)
+                return;
+
+            switch (type)
+            {
+                case SelectContainerStatus.Mapsets:
+                    if (selectedMapset == null)
```

Comment:
When can this be null?

Reply:
pretty sure when no map is selected, which can happen

Reply:
same goes for the exact same thing in the difficulty case

Reply:
Something tells me that when no map is selected the line above where you index the array is just gonna throw an exception.